recipes/root/all/conanfile.py

```diff
@@ -0,0 +1,305 @@
+                    "builtin_vdt": "ON",
+                    # No Conan packages available for these dependencies yet
+                    "davix": "OFF",
+                    "pythia6": "OFF",
+                    "pythia8": "OFF",
+                    "mysql": "OFF",
+                    "oracle": "OFF",
+                    "pgsql": "OFF",
+                    "gfal": "OFF",
+                    "tmva-pymva": "OFF",
+                    "xrootd": "OFF",
+                    "pyroot": self._pyrootopt,
+                    # Tell CMake where to look for Conan provided depedencies
+                    "CMAKE_LIBRARY_PATH": cmakelibpath,
+                    "CMAKE_INCLUDE_PATH": cmakeincludepath,
+                    # Configure install directories
+                    # Conan CCI hooks restrict the allowed install directory
+                    # names but ROOT is very picky about where build/runtime
+                    # resources get installed.
+                    # Set install prefix to work around these limitations
+                    # Following: https://github.com/conan-io/conan/issues/3695
+                    "CMAKE_INSTALL_PREFIX": "{}{}res".format(
+                        self.package_folder, os.sep
+                    ),
+                    # Fix some Conan-ROOT CMake variable naming differences
+                    "PNG_PNG_INCLUDE_DIR": ";".join(
+                        self.deps_cpp_info["libpng"].include_paths
+                    ),
+                    "LIBLZMA_INCLUDE_DIR": ";".join(
+                        self.deps_cpp_info["xz_utils"].include_paths
+                    ),
+                },
+            )
+        return self._cmake
+
+    def _move_findcmake_conan_to_root_dir(self):
+        for f in ["opengl_system", "GLEW", "glu", "TBB", "LibXml2", "ZLIB", "SQLite3"]:
+            shutil.copy(
+                "Find{}.cmake".format(f),
+                os.sep.join((self.source_folder, self._rootsrcdir, "cmake", "modules")),
+            )
+
+    @property
+    def _cmake_cxx_standard(self):
+        compileropt = self.settings.compiler.get_safe("cppstd")
+        if compileropt:
+            return str(compileropt)
+        else:
+            return "11"
+
+    @property
+    def _pyrootopt(self):
+        if self.options.python == PythonOption.OFF:
+            return "OFF"
+        else:
+            return "ON"
+
+    def build(self):
+        self._configured_cmake.build()
+
+    def package(self):
+        self._configured_cmake.install()
+        self.copy("LICENSE.txt", dst="licenses")
+        for dir in ["include", "lib", "bin"]:
+            os.symlink(
+                os.sep.join((self.package_folder, "res", dir)),
+                os.sep.join((self.package_folder, dir)),
+            )
+        # Fix for CMAKE-MODULES-CONFIG-FILES (KB-H016)
+        for cmakefile in glob(
+            os.sep.join((self.package_folder, "res", "cmake", "*Config*.cmake"))
+        ):
+            os.remove(cmakefile)
```

Comment:
there's a tool https://docs.conan.io/en/latest/reference/tools.html#tools-remove-files-by-mask to help with this

Reply:
Thanks. c4a2736 now uses this tool.